python/tvm/parser/__init__.py

```diff
@@ -24,4 +24,5 @@ def parse_expr(source):
     return _ffi_api.ParseExpr("string", source)
 
 def fromtext(source, source_name="from_string"):
+    # TODO(@tqchen): currently we have to invoke `str` which dramatically reduces performance.
```

Comment:
The C++ code was expecting a std::string, I changed it to tvm::String which fixes the overload but still incurs a copy. Copying can be very expensive here, for example if we have a model including the parameters it can be 100s of megabytes.

Reply:
Got it! Thanks for clarification. I will also have a check on it.